src/pages/PaymentDetails.tsx

```diff
@@ -297,7 +300,7 @@ export const PaymentDetails = () => {
                       </Table.BodyCell>
                       <Table.BodyCell textAlign="right">
                         <span className="Table-v2__cell--secondary">
-                          {formatDateTime(h.updatedAt)}
+                          {formatDateTimeWithSeconds(h.updatedAt)}
```

Comment:
Do we need to change the `Created At` as well?

Reply:
I really don't know, but I'll do so. If needed, we just have to change the function.